docs/tutorials/first-run-experience-tutorial.md

```diff
@@ -0,0 +1,152 @@
+---
+title: First-run experience tutorial
+description: Learn how to implement a first-run experience for your Office Add-in.
+ms.date: 08/09/2024
+ms.topic: how-to
+ms.localizationpriority: medium
+---
+
+# Build an Office Add-in with a basic first-run experience
+
+In this article, you'll walk through the process of updating a task pane add-in to include a [first-run experience](../design/first-run-experience-patterns.md) using the value placemat pattern. When the user runs the add-in, the add-in determines whether or not to show the first-run experience by checking [local storage](../develop/persisting-add-in-state-and-settings.md#browser-storage) for the existence of a certain flag.
+
+This tutorial provides instructions and screenshots for Excel but you can use a similar pattern to implement a first-run experience in other Office applications where Office Web Add-ins are supported. Also note that actual interactions with your selected Office application will differ.
+
+## Prerequisites
+
+1. Select the quick start for the Office application you'd like to use. Make sure to choose the Yeoman generator version.
+
+    - [Excel](../quickstarts/excel-quickstart-jquery.md)
+    - [OneNote](../quickstarts/onenote-quickstart.md)
+    - [Outlook](../quickstarts/outlook-quickstart.md)
+    - [PowerPoint](../quickstarts/powerpoint-quickstart.md)
+    - [Project](../quickstarts/project-quickstart.md)
+    - [Word](../quickstarts/word-quickstart.md)
+
+1. Follow the instructions in your selected quick start. After you complete its "Try it out" section, return here to continue.
+
+## Implement the first-run experience
+
+### Update the HTML file
+
+Be clear about the area of the UI that will be part of the first-run experience. In this tutorial, you'll create a div with `id` named "first-run-experience" that represents what users see only the first time they run your add-in.
+
+1. Open the **taskpane.html**. Replace the `<main>` element with the following markup, then save the file. Note about this markup:
```

Comment:
Since there is more than one note, consider pluralizing the lead-in.
```suggestion
1. Open the **taskpane.html**. Replace the `<main>` element with the following markup, then save the file. Some notes about this markup:
```

Reply:
@AlexJerabek We may need to clean this up because I got this terminology from elsewhere in the docs